layouts/funding_calls/list.html

```diff
@@ -86,9 +97,9 @@ <h5>${item.title}</h5>
     let fundingHTML = "";
     try {
       const dcCalls = await getDCFundingCalls("https://anubis.scilifelab.se/api/calls/open");
-      fundingHTML += getFundingsCallsHTML(fundings.calls);
+      fundingHTML += getFundingsCallsHTML(dcCalls);
```

Comment:
I am sorry, i did not noticed while suggesting. should not it be like:
```suggestion
      fundingHTML += getFundingsCallsHTML(dcCalls.calls);
```

Reply:
No `getDCFundingCalls` function returns only the list of fundings not the whole JSON ^^